arviz/plots/ppcplot.py

```diff
@@ -180,21 +188,40 @@ def plot_ppc(
     if data_pairs is None:
         data_pairs = {}
 
+    if backend is None:
+        backend = rcParams["plot.backend"]
+    backend = backend.lower()
+
     if animation_kwargs is None:
         animation_kwargs = {}
     if platform.system() == "Linux":
         animation_kwargs.setdefault("blit", True)
     else:
         animation_kwargs.setdefault("blit", False)
 
-    if animated and backend == "bokeh":
-        raise TypeError("Animation option is only supported with matplotlib backend.")
-
-    if animated and animation_kwargs["blit"] and platform.system() != "Linux":
-        _log.warning(
-            "If you experience problems rendering the animation try setting"
-            "`animation_kwargs({'blit':False}) or changing the plotting backend (e.g. to TkAgg)"
-        )
+    if animated:
+        if backend == "matplotlib":
```

Comment:
This logic should be moved into the backends themselves, so the higher level plotting function can stay unopionated. 

Let me know if you want me to edit PR. Happy to do so if you like!